activestorage/app/controllers/active_storage/blobs/redirect_controller.rb

```diff
@@ -10,7 +10,7 @@ class ActiveStorage::Blobs::RedirectController < ActiveStorage::BaseController
   include ActiveStorage::SetBlob
 
   def show
-    expires_in ActiveStorage.service_urls_expire_in
+    ActiveStorage.cache_service_urls ? expires_in(ActiveStorage.service_urls_expire_in) : no_store
```

Comment:
Any reason to use `:no_store` here?
```suggestion
    ActiveStorage.cache_service_urls && expires_in(ActiveStorage.service_urls_expire_in)
```

Reply:
@MatheusRich 
- without no_store, the default behavior will be max-age=0 the same as `expires_in(0)` which cache data but revalidate it immediately
- with no_store the data will not be cached at all

and this is the situation I faced before that made me think this way 
> This Pull Request has been created because some browsers have a bug and didn't respect the cache max-age and never invalidate the cached URLs even if service_urls_expire_in is set to zero. Hence, I had to override the redirect controller and disable the cache.
 This PR makes it easy to toggle caching without override and

I'm not entirely sure about this way, what do you think?

Reply:
I'm confused about the code. If `ActiveStorage.cache_service_urls` is false, you are just retuning a `:no_store` symbol. Does that really do anything?

Maybe the code should be
```suggestion
    expires_in(ActiveStorage.cache_service_urls ? ActiveStorage.service_urls_expire_in : no_store)
```

Reply:
Sorry. I misread it! Disregard the review